css/selectors/grouping.json

```diff
@@ -0,0 +1,55 @@
+{
+  "css": {
+    "selectors": {
+      "type": {
```

Comment:
```suggestion
      "list": {
```

Reply:
See: https://github.com/mdn/browser-compat-data/pull/532/files

It should be`type` not `list`.

Reply:
Done.